x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -98,7 +127,51 @@ const navigationTree: NavigationTreeDefinition = {
   footer: [
     {
       type: 'navGroup',
-      ...getPresets('management'),
+      id: 'projest_settings_project_nav',
+      title: 'Project settings',
+      icon: 'gear',
+      defaultIsCollapsed: true,
+      breadcrumbStatus: 'hidden',
+      children: [
+        {
+          id: 'settings',
+          children: [
+            {
+              link: 'monitoring',
+              title: i18n.translate('xpack.serverlessObservability.nav.mngt', {
+                defaultMessage: 'Management',
+              }),
+            },
+            {
+              link: 'integrations',
+            },
+            {
+              link: 'fleet',
+            },
+            {
+              id: 'users_and_roles',
```

Comment:
The external links are not defined yet. I added the default `https://cloud.elastic.co` but we can remove them until we have the links

Reply:
I'm fine with it but maybe it'd be good to have an issue to not forget about this, wdyt?